src/@next/components/atoms/Icon/definitions.ts

```diff
@@ -60,6 +62,16 @@ export const icons = {
       fill: "#21125e",
     },
   ],
+  tick: [
```

Comment:
After converting to path and fitting to 32x32 viewbox (using IcoMoon) the path is following:
`d: "M31.416 2.142l-19.195 29.858-11.638-12.414 2.889-2.709 8.164 8.708 16.448-25.586 3.331 2.142z"`